qi-lib/threading.rkt

```diff
@@ -24,7 +24,7 @@
    #'(on ags (~> clause ...))])
 
 (define-syntax-parser ~>>
-  [(_ (arg0 arg ...+) (~or (~datum sep) (~datum △)) clause:clause ...)
+  [(_ (arg0 arg ...+) (~or* (~datum sep) (~datum △)) clause:clause ...)
```

Comment:
Thanks for these `or*` changes - I did learn recently that `~or` is somewhat deprecated.